README.md

```diff
@@ -83,6 +84,7 @@ No modules.
 | <a name="input_dns_records_public_hosted_zone"></a> [dns\_records\_public\_hosted\_zone](#input\_dns\_records\_public\_hosted\_zone) | A list of DNS public (public hosted zone) records to create with the instance's IP | `list(any)` | `[]` | no |
 | <a name="input_ebs_block_device"></a> [ebs\_block\_device](#input\_ebs\_block\_device) | Additional EBS block devices to attach to the instance | `list(map(string))` | `[]` | no |
 | <a name="input_ebs_optimized"></a> [ebs\_optimized](#input\_ebs\_optimized) | Enable EBS Optimized | `string` | `"false"` | no |
+| <a name="input_enable_ssm_access"></a> [enable\_ssm\_access](#input\_enable\_ssm\_access) | If true, attaches SSM policy to instance role | `string` | `"false"` | no |
```

Comment:
@fgauchat-binbash FYI Terraform Docs should take care of updating this according to the changes I suggested.

Reply:
@diego-ojeda-binbash you're right. @fgauchat-binbash consider following and executing from your workstation this steps:

1. Have re-commit binary installed in your local machine
2. At the root context level `make init-makefiles`
3. At the root context level `make pre-commit` 
    1. in case some check fail, run it again since they update the files at the 1st run.